webapp/src/main/java/io/github/microcks/web/RestController.java

```diff
@@ -143,8 +143,8 @@ public ResponseEntity<?> execute(
                if (operation.getResourcePaths() != null) {
                   // Produce a matching regexp removing {part} and :part from pattern.
                   String operationPattern = getURIPattern(operation.getName());
-                  operationPattern = operationPattern.replaceAll("\\{.+\\}", "(.)+");
-                  operationPattern = operationPattern.replaceAll("(/:[^:^/]+)", "\\/(.+)");
+                  operationPattern = operationPattern.replaceAll("\\{.+\\}", "([^/])+");
+                  operationPattern = operationPattern.replaceAll("(/:[^:^/]+)", "([^/])+");
```

Comment:
I think this one is not OK as it removes first /.
`operationPattern = operationPattern.replaceAll("(/:[^:^/]+)", "\\/([^/]+)");` is the way to go. COuld you please modify this one ?